### DISCUSSION TITLE
Librerías en desusos (ya no es compatible)

### DISCUSSION THREAD

**Discussion by stexrbnblack** [codigoencasa/builderbot#767] [General]
Buenas tardes compañeros, No se si sea solo a mi oh pero últimamente ah ávido **conflicto con algunas librerías** del proyecto ocasionando dificultad al generar el QR y al leer lo con el cell.

aun ejecutando npm audit fix --force y dicha solución genera conflicto con el código.

algunas librerias que eh visto con problema por **compatible**:

- npm WARN en desuso **har-validator@5.1.5**: esta biblioteca ya no es compatible.
- npm WARN obsoleto **uuid@3.4.0**: actualice a la versión 7 o superior. Las versiones anteriores pueden usar Math.random() en ciertas circunstancias, lo que se sabe que es problemático. Ver https://v8.dev/blog/math-random para más detalles.
- npm WARN obsoleto **request@2.88.2**: la solicitud ha quedado obsoleta, consulte https://github.com/request/request/issues/3142.

**Comment by NicoPauer** [comment]
Talvez no genera QR actualizado al usar un estándar de código QR que ya no se usa
